src/timers/darwin.jl

```diff
@@ -0,0 +1,43 @@
+const RUSAGE_SELF = Cint(0)
+const RUSAGE_CHILDREN = Cint(-1)
+
+struct timeval
```

Comment:
This and `rusage` should use the standard Julia naming convention of CamelCase for types. So this should be `TimeVal` and the other should be `RUsage`.